be2-scala/docs/README.md

```diff
@@ -303,7 +305,26 @@ final case class DbActorReadLaoDataAck(laoData: LaoData) extends DbActorMessage
 
 For the Social Media functionality, each user has their own channel with the identifier `root/lao_id/own_pop_token` and each broadcast containing the message_id of a post will be written to `root/lao_id/posts`.
 
+For the Election functionality, we need to have a key pair stored safely somewhere so that we can encrypt/decrypt messages. That is why we use a `ELectionData` object to store the key pairs for the corresponding election.
```

Comment:
Typo in `ELectionData` -> `ElectionData` 😄

Reply:
Oops thank you for pointing it out